README.md

```diff
@@ -0,0 +1,27 @@
+# LFP Appeals Frontend
+This repo contains the frontend code for the LFP appeals service. It's currenty `WIP`.
+
+## Technologies
+
+- [NodeJS](https://nodejs.org/)
+- [expressJS](https://expressjs.com/)
+- [NunJucks](https://mozilla.github.io/nunjucks)
+- [gulpJS](https://gulpjs.com/)
+
+## Recommendations
+
+We recommend the use of Visual Studio Code for development as it allows the installation of the TSLint and the Nunjucks plugins. These plugins will make linting of TS and Nunjuck much better than mmost code editors.
+
+IntelliJ does not have a Nunjuck plugin
```

Comment:
True, although it does have other plugins like Twig which are mostly compatible with Nunjucks.

Technically I do not mind you saying this cause it is truth.

Some people might choose IntelliJ so we need to make sure long term that formatting is same across IDEs.

Reply:
I choose IntelliJ 😁